package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "browserify": "^13.0.0",
     "d3": "3.5",
+    "lodash.result": "^4.4.0",
```

Comment:
@gordonwoodhull good catch on ```devDependencies```.

As for there being a runtime dependency, the way browserify is currently configured ```lodash.result``` will get rolled into the compiled ```crossfilter.js``` and ```crossfilter.min.js``` files during build. It won't need to be included separately.